drop-transfer/src/ws/client/mod.rs

```diff
@@ -142,7 +142,11 @@ async fn make_request(
     auth: &auth::Context,
     logger: &slog::Logger,
 ) -> Result<(), tungstenite::Error> {
-    let url = format!("ws://{ip}:{}/drop/{version}", drop_config::PORT);
+    let addr = SocketAddr::new(ip, drop_config::PORT);
+
+    let url = format!("ws://{addr}/drop/{version}",);
+
+    debug!(logger, "Making HTTP request: {url}");
```

Comment:
nitpick: if drop-transfer is agnostic to IPv4/6 maybe it's worth extracting it into FFI layer and pass it down here

Reply:
Um, sorry but I don't get it. What do you want to move to ffi layer? the URL construction?

Reply:
Address construction

Reply:
So you want to pass the `SocketAddr` from the ffi layer. It actually makes sense. This way we would detach `drop-transfer` crate from the concrete port.
Let me do this one in Monday

Reply:
@LukasPukenis As it turns out it's not an easy task, because we construct the address from the `Transfer` object which cannot store port information as it is not stable per peer